From 2082cdf230574fa7482903730b584445ed3a2a08 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 27 Jul 2015 13:30:22 +0200
Subject: [PATCH] crypt: s/allow-discards/discard for crypttab

actually the option in crypttab is named "discard" and not
"allow-discards"

https://bugzilla.suse.com/show_bug.cgi?id=932972
---
 modules.d/90crypt/crypt-run-generator.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/modules.d/90crypt/crypt-run-generator.sh b/modules.d/90crypt/crypt-run-generator.sh
index 5bf60ec..8dad76b 100755
--- a/modules.d/90crypt/crypt-run-generator.sh
+++ b/modules.d/90crypt/crypt-run-generator.sh
@@ -15,10 +15,10 @@ if strstr "$(cryptsetup --help)" "allow-discards"; then
     if discarduuids=$(getargs "rd.luks.allow-discards"); then
         discarduuids=$(str_replace "$discarduuids" 'luks-' '')
         if strstr " $discarduuids " " ${luks##luks-}"; then
-            allowdiscards="allow-discards"
+            allowdiscards="discard"
         fi
     elif getargbool 0 rd.luks.allow-discards; then
-        allowdiscards="allow-discards"
+        allowdiscards="discard"
     fi
 fi
 
